doc/ref_model/chapters/chapter02.md

```diff
@@ -290,29 +292,29 @@ In **[Chapter 4](./chapter04.md)** these **B (Basic)** and **H (High) Performanc
-|---|---|---|---|---|
-|**High performance CPU**|❌|✅|Nodes that have predictable computing performance (or higher clock speed).|May use vanilla VIM/K8S scheduling instead.|
-|**High performance storage**|❌|✅|Nodes that have low storage latency and/or high storage IOPS|
-|**High memory**|❌|✅|Nodes that have high amounts of RAM.|May use vanilla VIM/K8S scheduling instead.|
-|**GPU**|❌|✅|denotes the presence of a consumable GPU on the node|May use Node Feature Discovery.|
-|**High speed network (25G)**|❌|✅|denotes the presence of network links (to the DC network) of speed of 25 Gbps or greater on the node.|
-|**High speed network (100G)**|❌|✅|denotes the presence of network links (to the DC network) of speed of 100 Gbps or greater on the node.|
-|**100ms Latency**|✅|✅|labels a host/node as located in a site that has less than 100ms of latency to the specified set of users.|Cannot practically enforce latency to weakly defined users. Requires defining the users/locations this refers to.|
-|**50ms Latency**|✅|✅|labels a host/node as located in a site that has less than 50ms of latency to the specified set of users.|Cannot practically enforce latency to weakly defined users. Requires defining the users/locations this refers to.|
-|**10ms Latency**|✅|✅|labels a host/node as located in a site that has less than 10ms of latency to the specified set of users.|Cannot practically enforce latency to weakly defined users. Requires defining the users/locations this refers to.|
-|**Low Latency** - Edge Sites|✅|✅|labels a host/node as located in an edge site, for workloads requiring low latency to final users or geographical distribution.|
-|**Fixed function accelerator**|❌|✅|labels a host/node that includes a consumable fixed function accelerator (non programmable, eg Crypto, vRAN-specific adapter).|
-|**Firmware-programmable adapter**|❌|✅|labels a host/node that includes a consumable Firmware-programmable adapter (programmable, eg Network/storage FPGA with programmable part of firmware image).|
-|**SmartNIC enabled**|❌|✅|labels a host/node that includes a Programmable accelerator for vSwitch/vRouter, Network Function and/or Hardware Infrastructure.|
-|**SmartSwitch enabled**|❌|✅|labels a host/node that is *connected to* a Programmable Switch Fabric or TOR switch|
-
->***Note:** This is an initial set of proposed profiles and flavours and it is expected that more profiles will be added as more requirements are gathered and as technology enhances and matures. For instance, the following profiles may be added in future releases:*
+<a name="2.4.2"></a>
+### 2.4.2 Profile Extensions (specialisations)
+
+Profile Extensions are meant to be used as labels for infrastructure, identifying the nodes that implement special capabilities that go beyond the profile baseline. Certain profile pxtensions may be relevant only for some profiles.
+The following **profile extensions** are proposed:
```

Comment:
@karinesevilla I think the intent is that even for Basic, we are able to specify an intent although there may not be a QoS -- it would be like a "should".

Reply:
Yes it is optional,but it complicates and it is not necessary for me.
If you think it's useful, let's keep it as it is and remove the suggestions